drivers/stepper/stepper_shell.c

```diff
@@ -69,7 +69,7 @@ static void print_callback(const struct device *dev, const enum stepper_event ev
 
 static bool stepper_device_check(const struct device *dev)
```

Comment:
Maybe change the function name to `device_is_stepper` to match the rest?

Reply:
yeah good idea, done